### ISSUE TITLE
Memory CGRA_INFO

### ISSUE THREAD

**Issue by cdonovick** [StanfordAHA/CGRAGenerator#49]
CGRA_info for fifo depth is not being generate correctly in dev
```
    <mem feature_address='4' data_bus='BUS16' control_bus='BUS1'>
       <mode bith='1' bitl='0'>00</mode>
       <tile_en bith='2' bitl='2'>0</tile_en>
       <depth bith='15' bitl='3'>0</depth>
       <almost_count bith='19' bitl='16'>0</almost_count>
       <chain_enable bith='20' bitl='20'>0</chain_enable>
    </mem>
```

should be 
```
    <mem feature_address='4' data_bus='BUS16' control_bus='BUS1'>
       <mode bith='1' bitl='0'>00</mode>
       <tile_en bith='2' bitl='2'>0</tile_en>
       <fifo_depth bith='15' bitl='3'>0</depth>
       <almost_count bith='19' bitl='16'>0</almost_count>
       <chain_enable bith='20' bitl='20'>0</chain_enable>
    </mem>
```

**Comment by steveri** [comment]
@cdonovick Hopefully you are not blocked on this.
Meanwhile I am implementing a quick fix, basically
`sed 's/depth/fifo_depth/' < old_cgra_info.txt > new_cgra_info.txt`
and I will check it into 'dev' so that you can go forward...

@alexcarsello  since this is not part of the PE I guess it officially defaults to you (and/or should we revisit this arrangement at some point?)  However, it's a very easy fix and I can do it if you like...

SR

**Comment by cdonovick** [comment]
Found another bug

    <mem feature_address='4' data_bus='BUS16' control_bus='BUS1'>
       <mode bith='1' bitl='0'>00</mode>
       <tile_en bith='2' bitl='2'>0</tile_en>
       <depth bith='15' bitl='3'>0</depth>
       <almost_count bith='19' bitl='16'>0</almost_count>
       <chain_enable bith='20' bitl='20'>0</chain_enable>
    </mem>

should be

    <mem feature_address='4' data_bus='BUS16' control_bus='BUS1'>
       <mode bith='1' bitl='0'>00</mode>
       <tile_en bith='2' bitl='2'>0</tile_en>
       <depth bith='15' bitl='3'>0</depth>
       <almost_full_count bith='19' bitl='16'>0</almost_count>
       <chain_enable bith='20' bitl='20'>0</chain_enable>
    </mem>

**Comment by alexcarsello** [comment]
Just fixed this. Changes pushed to dev.
